tox.ini

```diff
@@ -37,6 +37,7 @@ deps =
     mypy
 setenv = PYTEST_ADDOPTS = "--color=yes"
 commands =
+    pip install cftime==1.4.1
```

Comment:
Why not add this to the `deps` section?

Reply:
My goal was to limit this as much as possible. Xclim works with cftime 1.5.0, so no need to pin it, it's not "their" fault.

Reply:
I just meant you could have added `cf-time==1.4.1` to the deps section of the `doctest` overrides a few lines above. Both work though.